server/src/test/java/org/apache/iotdb/db/integration/IoTDBCloseIT.java

```diff
@@ -85,7 +86,7 @@ private static void insertData()
             .getConnection(Config.IOTDB_URL_PREFIX + "127.0.0.1:6667/", "root", "root");
          Statement statement = connection.createStatement()) {
 
-      for (String sql : Constant.create_sql) {
+      for (String sql : org.apache.iotdb.db.constant.TestConstant.create_sql) {
```

Comment:
```suggestion
      for (String sql : TestConstant.create_sql) {
```
And all the same appearance in this test class : )

Reply:
fixed